<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminView</class>
 <widget class="QDialog" name="adminView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>361</width>
     <height>461</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>359</width>
      <height>459</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>331</width>
       <height>441</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Post</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_bar_line_edit">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>40</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="title_search_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QComboBox" name="category_combo_box">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="rating_combo_box">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>480</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Save &amp; exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keywords">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>New Category</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CategoryLine">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
